vaadin-grid-flow-parent/vaadin-grid-flow/src/main/resources/META-INF/resources/frontend/gridConnector.js

```diff
@@ -667,6 +667,9 @@ import { isFocusable } from '@vaadin/grid/src/vaadin-grid-active-item-mixin.js';
           const firstPage = index / grid.pageSize;
           const updatedPageCount = Math.ceil(items.length / grid.pageSize);
 
+          const selections = [];
```

Comment:
Are these changes still needed? I assumed the addition of `grid._effectiveSize = grid._cache.effectiveSize;` fixes the issue.

Reply:
Removed the other changes